less/components/popups.less

```diff
@@ -9,7 +9,7 @@
 //--------------------------------------
 
 .popup-shadow {
-    filter: drop-shadow(0px 2px 4px rgba(0, 0, 0, 0.5));
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.5);
```

Comment:
yup, this happens in ff because of this: https://bugzilla.mozilla.org/show_bug.cgi?id=876341

makes any container that has a "transform" effect applied to if act as a container for fixed positioned elements instead of the viewport